Common/src/fem_geometry_structure.cpp

```diff
@@ -6929,4 +6863,66 @@ void CMeshFEM_DG::InitStaticMeshMovement(CConfig              *config,
     default:  /* Just to avoid a compiler warning. */
       break;
   }
+  
+  if (config->GetSurface_Movement(MOVING_WALL)){
+    /*--- Loop over the physical boundaries. Skip the periodic boundaries. ---*/
+    for(unsigned short i=0; i<boundaries.size(); ++i) {
+      if( !boundaries[i].periodicBoundary ) {
+        
+        /* Check if for this boundary a motion has been specified. */
+        if (config->GetMarker_All_Moving(i) == YES) {
+          
+          /* Determine the prescribed translation velocity, rotation rate
+               and rotation center. */
+          const su2double Center[] = {config->GetMotion_Origin()[0],
+                                      config->GetMotion_Origin()[1],
```

Comment:
If this is the general way in which these methods will be called, why not pass iDim as an argument and void returning pointers?

Reply:
Thanks for pointing that out. Thats something I wanted to change, thanks for reminding me :)